tfjs-backend-webgl/src/webgl_util.ts

```diff
@@ -395,30 +403,42 @@ export function getTextureShapeFromLogicalShape(
   }
 
   let size = util.sizeFromShape(logShape);
-  if (logShape.length <= 1 && size <= maxTexSize) {
-    return [1, size];
+  let textureShape: [number, number];
```

Comment:
nit: `let textureShape = null`. Then the last `else {}` can be removed?

Reply:
Done.